playbooks/pre-install-playbook/roles/viya-ark.preinstall/tasks/pre.proxy_check.yml

```diff
@@ -62,7 +62,7 @@
   - name: "Assert that https_proxy is set as environment variable"
     assert:
       that:
-        - ("{{ https_proxy }}" != "")
+        - ("https_proxy" != "")
```

Comment:
it looks like this would just do a string compare and always be true?

Shouldn't it be more like this:

```yaml
    assert:
      that:
        - (https_proxy != "")
```

Can you verify and confirm?

Reply:
You are correct.   I see other instances of the expression, in other tasks in the pre.proxy_check.yml play.

I am also seeing that there needs to be some setup done to more properly exercise this set of tasks, so will need to spend some time on that.

Reply:
Found a few more instances of jinja2 templating delimiter usage.